### PR TITLE
Added classroom website in the repository

### PR DESCRIPTION
**Pull request by RiyaGupta89**
Added the new classroom website project in the repository. Also created the readme.md file with the image. 
classroom

### CODE REVIEW

**Comment by chicken-biryani** [comment]
@RiyaGupta89 

Please make 1 PR per issue, per project, don't make different PRs, it leaves a bad impression.

File: CLASSROOM SCHEDULER WEBSITE/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+<p>A website that lets you write about all your classes at one place. You can also search by typing the name of the teacher if you have so many classes to take.</p>
+
+<h3>Used Technologies</h3>
+<ul>
+  <li>HTML5</li>
+  <li>CSS3</li>
+  <li>JavaScript</li>
+</ul>
+
```
Comment (chicken-biryani) [inline CLASSROOM SCHEDULER WEBSITE/README.md:11]: Add a "Use of this project" section over here, that describes places this project can be used.

File: CLASSROOM SCHEDULER WEBSITE/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+- Go to the directory
+- Run the index.html file
+- Schedule your new batch.
+- Add more and more batches.
+
+
+<h3> ScreenShots </h3>  
+<img src = "./classroom.png" alt="Sample Image">
```
Comment (chicken-biryani) [inline CLASSROOM SCHEDULER WEBSITE/README.md:27]: The image isn't working.

File: CLASSROOM SCHEDULER WEBSITE/index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+
+
+   
+              
+                
+            
+
```
Comment (chicken-biryani) [inline CLASSROOM SCHEDULER WEBSITE/index.html:65]: Remove extra lines.

File: CLASSROOM SCHEDULER WEBSITE/style.css [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    color: white;
+}
+
+.deleteBtn:hover {
+    background-color: rgb(204, 142, 9);
+}
+
+.wrapper input {
+    padding: 3px 15px;
+}
```
Comment (chicken-biryani) [inline CLASSROOM SCHEDULER WEBSITE/style.css:125]: Add a new line at the end of file.

**Comment by RiyaGupta89** [comment]
> @RiyaGupta89
> 
> Please make 1 PR per issue, per project, don't make different PRs, it leaves a bad impression.

Okay @chicken-biryani. I'll keep this in mind. Thankyou.

**Comment by RiyaGupta89** [comment]
@chicken-biryani Please review the code, I've done all the things you asked me to do.

**Review by Ayushparikh-code — APPROVED** [review]

**Comment by Ayushparikh-code** [comment]
> @chicken-biryani Please review the code, I've done all the things you asked me to do.

ok, update ur projects in the index.md too 
am merging this PR

**Pull request merged**